smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -460,8 +460,8 @@ void updateDevices() {
 			childDevice.sendEvent(name: "saturation", value: device.color.saturation * 100)
 		}
 		childDevice.sendEvent(name: "label", value: device.label)
-		childDevice.sendEvent(name: "level", value: Math.round((device.brightness ?: 1) * 100))
-		childDevice.sendEvent(name: "switch.setLevel", value: Math.round((device.brightness ?: 1) * 100))
+		childDevice.sendEvent(name: "level", value: Math.round((device.brightness != null ? device.brightness : 1) * 100))
```

Comment:
Original lines means the same as what you changed it to. It is just  a short form.

Reply:
Not quite true. 0 has groovy truth of false which should not be defaulted to 1 (full brightness)